packages/application-shell-connectors/src/components/application-context/application-context.tsx

```diff
@@ -0,0 +1,200 @@
+import React from 'react';
+import moment from 'moment-timezone';
+import getDisplayName from '../../utils/get-display-name';
+
+type AdditionalProperties = { [key: string]: unknown };
+type TRawUser = {
```

Comment:
To avoid naming conflicts, I decided to prefix most of the types here with `T`.

Reply:
Also, the "raw" user is the one we get from the API. Hopefully the name makes sense

Reply:
I often postfixed typed with Type. E.g PermissionsType. I don‘t really mind which way we go as long as we all pick the same 🤣